draft-ietf-quic-transport.md

```diff
@@ -3885,14 +3885,17 @@ the connection ID as part of its token validation logic (see
 The next Initial packet from the client uses the connection ID and token values
 from the Retry packet (see {{negotiating-connection-ids}}).  Aside from this,
 the Initial packet sent by the client is subject to the same restrictions as the
-first Initial packet.  A client can either reuse the cryptographic handshake
-message or construct a new one at its discretion.
+first Initial packet.  A client MUST use the same cryptographic handshake
+message it includes in this packet.  A server MAY treat a packet that
+contains a different cryptographic handshake message as a connection or discard
```

Comment:
```suggestion
contains a different cryptographic handshake message as a separate connection or discard
```

Reply:
Oops, I missed a word, but it wasn't that one.